repl/README.md

```diff
@@ -0,0 +1,77 @@
+# CEL REPL
+
+This is a simple tool for experimenting with CEL expressions and learning the syntax.
+
+## Usage
+The REPL (Read Evaluate Print Loop) is implemented as a command line tool.
+
+By default, the the input will be interpreted as a cel expression to evaluate.
+
+Special commands (prefixed with '%') are used to update the evaluation environment.
+
+An example session:
+
+```
+# from a cel-go clone
+$ cd ./repl
+
+$ go run .
+CEL REPL
+%exit or EOF to quit.
+
+cel-repl> %let x = 10
+cel-repl> %let y = {'abc': {'def': [1, 2, 3]}}
+cel-repl> y.abc.def.filter(el, el < x)
+[1 2 3] : list(int)
+cel-repl> %delete x
+cel-repl> y.abc.def.filter(el, el < x)
+Expr failed:
+ERROR: <input>:1:27: undeclared reference to 'x' (in container '')
+ | y.abc.def.filter(el, el < x)
+ | ..........................^
+cel-repl> %declare x : int
+cel-repl> y.abc.def.filter(el, el < x)
+Expr failed:
+no such attribute: id: 8, names: [x]
+no such attribute: id: 8, names: [x] : list(int)
+cel-repl> y.abc.def.filter(el, el < x || el > 0)
+[1 2 3] : list(int)
+cel-repl> %exit
+```
+
+### Commands
+
+*%let* introduce or update a variable declaration and provide a definition (as another CEL expression). A type hint is optionally provided to check that the provided expression has the expected type. `%let <identifier> (: <type>)? = <expr>`
```

Comment:
As a minor nit, please cap the length of markdown lines to 80 characters.

Reply:
done.